web/src/shared/components/ourdna/TableTile.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react'
+import { Card, Table } from 'semantic-ui-react'
+
+interface TableTileProps {
+    header: string
+    data: Record<string, string | number>
+    columns: Array<string>
+    icon: React.ReactNode
+}
+
+const TableTile: React.FC<TableTileProps> = ({ header, data, columns, icon }) => (
+    <Card
+        fluid
+        style={{
+            backgroundColor: 'var(--color-bg-card)',
+            boxShadow: 'rgba(0, 0, 0, 0.24) 0px 3px 8px',
```

Comment:
just a nit, but do you think we should keep it consistent and pull the shadow as a var defined in the css